src/tests/databases/core_9/meta.txt

```diff
@@ -14,11 +14,12 @@
 2	1	species.taxonomy_id	6239
 10	1	species.type	monsters
 5	1	species.url	Jabbe
-17	1	genebuild.version	EXT01
+17	1	genebuild.version	ENS01
 18	1	genebuild.sample_gene	ENSAMXG00005000318
 19	1	genebuild.sample_location	KB871578.1:9766653-9817473
 20	1	strain.type	test
 23	1	genome.genome_uuid	a733550b-93e7-11ec-a39d-005056b38ce3
 24	1	genebuild.havana_datafreeze_date	test2
 25	1	assembly.total_genome_length	546
-26	1	genebuild.start_date	test
\ No newline at end of file
+26	1	genebuild.start_date	test
```

Comment:
Start date should be a date

Reply:
Replaced with 2023-07-Ensembl in all cases